lib/Conversion/TorchToTosa/TorchToTosa.cpp

```diff
@@ -4197,9 +4197,13 @@ class ConvertAtenConstPatternOp : public OpConversionPattern<AtenOpT> {
 
     // FIXME: Handle layout, device and pin_memory. Assume dtype has been
     // processed to set output type correctly?
-    if (!op.getLayout().getType().template isa<Torch::NoneType>())
+    int64_t layout;
+    if (!op.getLayout().getType().template isa<Torch::NoneType>() &&
+        (!matchPattern(op.getLayout(), m_TorchConstantInt(&layout)) ||
+         layout != 0)) {
```

Comment:
Looking at similar code in TorchOps.cpp, can we check against torch_upstream::Layout::Strided instead of 0? It makes it easier to read and keeps consistent in both locations, which could save us from unexpected failures if torch changes.